<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edytor</class>
 <widget class="QWidget" name="edytor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>584</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edytor pojęc</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="Layout" stretch="1,0">
     <item>
      <layout class="QHBoxLayout" name="pojecieLayout" stretch="1,1">
       <item>
        <layout class="QVBoxLayout" name="terminyLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>termin preferowany:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="preferowanyQline"/>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QListWidget" name="ukryteQlista">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
             </property>
             <property name="viewMode">
              <enum>QListView::ListMode</enum>
             </property>
             <property name="selectionRectVisible">
              <bool>false</bool>
             </property>
             <property name="currentRow">
              <number>-1</number>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>Nowy element</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QListWidget" name="alternatywneQlista">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
             </property>
             <property name="viewMode">
              <enum>QListView::ListMode</enum>
             </property>
             <property name="selectionRectVisible">
              <bool>false</bool>
             </property>
             <property name="currentRow">
              <number>-1</number>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>Nowy element</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>terminy ukryte:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>terminy alternatywne:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="pojeciaLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>definicja pojęcia:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="definicjaText"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>przykłady użycia</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="przykladyText"/>
           </item>
           <item>
            <widget class="QPushButton" name="zob_uwagiButton">
             <property name="text">
              <string>zobacz uwagi edytorskie</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="relacjeLayout">
       <item>
        <layout class="QVBoxLayout" name="szerszeLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Pojęcie szersze</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="SzerszeQlista">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="zmienButton">
           <property name="text">
            <string>zmień</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="wezszeLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Pojęcia węższe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="wezszeQlista"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="dodaj_wezszeButton">
             <property name="text">
              <string>dodaj</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="usun_wezszeButton">
             <property name="text">
              <string>usuń</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="skojarzoneLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Pojęcia skojarzone</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="skojarzoneQlista"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="dodaj_skojarzoneButton">
             <property name="text">
              <string>dodaj</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="usun_skojarzoneButton">
             <property name="text">
              <string>usuń</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="zatwierdzButton">
     <property name="text">
      <string>ZATWIERDŹ</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
